pkgs/tools/networking/modemmanager/default.nix

```diff
@@ -74,6 +68,7 @@ stdenv.mkDerivation rec {
     "--sysconfdir=/etc"
     "--localstatedir=/var"
     "-Dvapi=true"
+    "-Dexamples=false"
```

Comment:
As discussed in https://github.com/NixOS/nixpkgs/pull/214086#issuecomment-1427115102, I do not think we want that.

Reply:
Ah, I didn't read the full discussion there. I set examples to false because the patch I dropped here essentially achieved the same anyway, as far as I understood it.

Reply:
The flag also prevents examples from being installed to `$out/share`. The patch just stops an attempt on creating an empty directory in `/etc`.

Reply:
Ah, yeah, I see that. I'll revert that change and push it to be a clean bump.